.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@ Bug fixes:
 
 Enhancements:
 - for Traefik v2: we only accept bug fixes
-- for Traefik v3: use branch master
+- for Traefik v3: use branch v3.0
```

Comment:
The template should not be modified in this PR, also iirc, the templates in use are those of the master branch.

```suggestion
- for Traefik v3: use branch master
```